packages/theme-chalk/src/col.scss

```diff
@@ -12,7 +12,8 @@
 
 @for $i from 0 through 24 {
   .el-col-#{$i} {
-    width: (1 / 24 * $i * 100) * 1%;
+    max-width: (1 / 24 * $i * 100) * 1%;
+    flex: 0 0 (1 / 24 * $i * 100) * 1%;
```

Comment:
Adding flex seems weird since this is not a child of `flex` displayed HTML Element

Reply:
I refer to other frameworks, they also use `flex`.

Reply:
So could you give me a screen shot on this?

Reply:
We closed testing deploy for saving the build time.. So I cannot preview the changes. Thank you

Reply:
Yes, I also create a preview website, I will delete it after the pr is closed.
![image](https://user-images.githubusercontent.com/38392315/101564993-18315380-3a07-11eb-81f2-bc24c31e93f4.png)

https://test.kooriookami.top/deploy-preview-875--element-plus/#/zh-CN/component/layout

Reply:
LGTM